tests/time_tests/src/timetests/timetest_infer.cpp

```diff
@@ -45,10 +49,14 @@ int runPipeline(const std::string &model, const std::string &device) {
             cnnNetwork = ie.ReadNetwork(model);
             batchSize = cnnNetwork.getBatchSize();
           }
-
           {
             SCOPED_TIMER(load_network);
-            exeNetwork = ie.LoadNetwork(cnnNetwork, device);
+            if (isVPUCompilerMLIR) {
+                exeNetwork = ie.LoadNetwork(cnnNetwork, device, {{"VPUX_COMPILER_TYPE", "MLIR"}});
+            }
+            else {
+                exeNetwork = ie.LoadNetwork(cnnNetwork, device);
```

Comment:
This differs with previous impl of timetest_infer_cache. It skips ReadNetwork stage and uses another call for network compilation: ie.LoadNetwork(<path_to_model>, device); By the [doc](https://docs.openvino.ai/latest/openvino_docs_IE_DG_Model_caching_overview.html#even-faster-use-loadnetwork-modelpath) such implementation should provide better FIL metric. 
I am also not sure that VPU MLIR version is implemented in the expected way. @vinograd47 could you please comment?

Reply:
Not sure if I understood your question. As far as I understand, this `LoadNetwork` form is used to enable caching mechanism. The caching mechanism is implemented on the core part level and uses import/export functionality of the plugin. VPU plugin supports this functionality independently from the underlying compiler. So, it should work in any way.

Reply:
There are [two ways](https://docs.openvino.ai/latest/openvino_docs_IE_DG_Model_caching_overview.html#model-caching-overview) how model caching can be implemented from API perspective:
1. 
`auto cnnNet = ie.ReadNetwork(modelPath);`
`ie.LoadNetwork(cnnNet, device, deviceConfig);`
2.
`ie.LoadNetwork(modelPath, device, deviceConfig);`
Second one shows better performance since it skips redundant ReadNetwork call and it looks preferable for CPU model caching benchmarking cause we share this data with customers. The question to you: which way should be used for VPU tests?

Reply:
@ArtemySkrebkov-intel @ilya-lavrenov Could you please help to answer this question? 

As I can see, VPU plugin implements internal `LoadNetwork`, which takes `CNNNetwork`. If the OpenVINO core can redirect both variants to this internal call, then I think it doesn't matter from the VPU plugin point of view.

Reply:
Doesn't it already work this way? I mean redirecting to internal function which takes `CNNNetwork`?

Reply:
I don't see that something has changed here. Before and after code uses `LoadNetwork(cnnNetwork)`, so what is the difference? 

About implementation itself:
1. `core.LoadNetwork(filename)` reads model and calls `plugin.LoadNetwork(cnnNetwork)`. This allows during caching skip read models on next iterations. Hash is computed based on `filename`
2. `core.LoadNetwork(cnnNetwork)` calls `plugin.LoadNetwork(cnnNetwork)`. Hash is computed on each iterations on `cnnNetwork` which is time consuming, so N1 approach is more preferable